frontend/src/AlertsManagement/History.js

```diff
@@ -25,54 +47,68 @@ class History extends Component {
     });
   }
 
-  createListItems = (allAlerts) => {
-    let alertItems = [];
-    allAlerts.forEach((alert, alertId) => {
-      alertItems.push(
-        <ListItem key={alertId} role={undefined} divider 
-          onClick={() => this.props.history.push(`alerts/${alertId}`)}
-        >
-          <ListItemText id={alertId}
-            disableTypography
-            primary={<Typography variant="body1" >Alert on 
-              <Box fontWeight='fontWeightBold' display='inline' m={1} style={{ color: '#0FA3B1'}}>
-                {` ${alert.timestampDate} (${formatDate(alert.timestampDate)})`}
-              </Box> has
-              <Box display='inline' m={1} style={{ color: '#0FA3B1'}}>
-                {` ${alert.anomalies.length} anomalies`}
-              </Box> with status
-              <Box display='inline' m={1} style={{ color: '#0FA3B1'}}>
-                {` ${alert.status}`}
-              </Box>
-              </Typography>
-            }
-          />
-        </ListItem>
-      );
-    });
-    return alertItems;
-  }
-
   render() {
     const styles = ({
       root: {
         flexGrow: 1,
-        maxWidth: 900,
+        maxWidth: '90%',
         margin: 'auto',
       },
+      header: {
+        marginTop: '50px',
+        marginBottom: '30px'
+      },
+      chip: {
+        margin: '5px',
+      }
     });
+
     const { allAlerts } = this.state;
     
-    if (allAlerts === null) {
-      return <div />;
-    }
+    if (allAlerts === null) return <div />;
 
     return (
       <div style={styles.root}>
-        <Typography variant="h4" gutterBottom style={{ marginTop: '50px'}}>Alerts Archive</Typography>
-        <List className="all-alerts-list">
-          {this.createListItems(allAlerts)}
-        </List>
+        <Typography variant="h4" gutterBottom style={styles.header}>Alerts Archive</Typography>
+        
+        <MaterialTable icons={tableIcons}
+          actions={[
+            {
+              icon: () => <LinkIcon />,
+              tooltip: 'See More',
+              onClick: (_event, rowData) => this.props.history.push(`alerts/${rowData.id}`),
+            }
+          ]}
+          columns={[
+            { title: 'Timestamp', field: 'timestampDate', filtering: false },
+            { title: '# of Anomalies', field: 'numAnomalies' },
```

Comment:
does this show up on the archive page?
I don't see a column for the `# of Anomalies`...

Reply:
Oops, I took an old screenshot. Updated with new screenshot